File: app/src/main/java/com/example/hackturin/BroadcastReceiverCustom.java
package com.example.hackturin;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class BroadcastReceiverCustom extends BroadcastReceiver {

    @Override
    public void onReceive(Context context, Intent intent) {
        double latitude = Double.valueOf(intent.getStringExtra("latutide"));
        double longitude = Double.valueOf(intent.getStringExtra("longitude"));

    }
}

File: app/src/main/java/com/example/hackturin/service/BroadcastReceiverCustom.kt
package com.example.hackturin.service

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import org.koin.core.KoinComponent

class BroadcastReceiverCustom : BroadcastReceiver(), KoinComponent {

    override fun onReceive(context: Context, intent: Intent) {
        val latitude = intent.getStringExtra("latutide").toDouble()
        val longitude = intent.getStringExtra("longitude").toDouble()

        val broadcastIntent = Intent()
        broadcastIntent.putExtra("LATITUDE", latitude)
        broadcastIntent.putExtra("LONGITUDE", longitude)
        context.sendBroadcast(broadcastIntent)
    }
}
